docs/docs/docs/05-utils/pack-up/00-intro.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Introduction
+tags:
+  - pack-up
+  - bundling
+  - packages
+---
+
+## What is pack-up?
+
+Pack-Up is primarily a CLI interface to develop & build CJS/ESM interopable packages. It's built on top of [vite](https://vitejs.dev/),
```

Comment:
comming from a lack of understanding of bundling and transpiling, what interopable packages mean? packages that both can be imported from cjs and esm projects?

Reply:
yes!

Reply:
I could add a link to the meaning?

Reply:
yes! that would help!

Reply:
updated ✅